### ISSUE TITLE
Support the ability to start an Ingester even if others aren't ready

### ISSUE THREAD

**Issue by joe-elliott** [grafana/tempo#918]
**Is your feature request related to a problem? Please describe.**
Currently an ingester will fail to become ready if another ingester is unhealthy in the ring. It logs this:

```
found an existing instance(s) with a problem in the ring, this instance cannot become ready until this problem is resolved. The /ring http endpoint on the distributor (or single binary) provides visibility into the ring.
```

**Describe the solution you'd like**
Provide a config option to move past this error and completely join the ring and become ready even if there are unhealthy ring members.

**Describe alternatives you've considered**

**Additional context**
This will require changes in upstream Cortex.

**Comment by joe-elliott** [comment]
This was fixed when we moved to the `dskit/ring` in #1100.

An additional parameter `readiness_check_ring_health` was added that defaults to `true`. If the behavior described in this issue is desired then set this to `false`.

```
ingester:
  lifecycler:
    readiness_check_ring_health: true
```

**Comment by rmn-lux** [comment]
also try this

```
ingester:
      autoforget_unhealthy: true

```
